make_deb

```diff
@@ -76,7 +76,7 @@ package_name="foglamp-gui-${version}"
 # Print the summary of findings
 echo "The package root directory is         : ${GIT_ROOT}"
 echo "The FogLAMP gui version is            : ${version}"
-echo "The Package will be built in          : ${BUILD_ROOT}"
+echo "The package will be built in          : ${BUILD_ROOT}"
```

Comment:
O/P fixes as per ^^ text changes in https://github.com/foglamp/foglamp-gui/blob/develop/debian-readme.md